### ISSUE TITLE
Building shows an issue but not explanation available

### ISSUE THREAD

**Issue by Ecostack** [mlange-42/tiny-world#210]
As you can see in the screenshot, the building has the exclamation mark but it shows no other information on why this is the case.

<img width="934" alt="Screenshot 2024-03-14 at 21 29 21" src="https://github.com/mlange-42/tiny-world/assets/8505607/a8c5554c-099a-4c2a-9be2-f34735457d85">

**Comment by mlange-42** [comment]
Yeah, there is generally no information about exclamation marks. They simply show that there is zero production, for various reasons:

- No usable tiles nearby
- No road access (in your case)
- Storage full due to too far from or not connected to warehouse

Not sure how information about the cause of the problem could be visualized. Maybe by having different warning symbols instead of the exclamation mark? Or put a little icon next to/above the exclam?

**Comment by mlange-42** [comment]
BTW, the mill should also have a warning, as it has no road. Fixed that on #211.

**Comment by Ecostack** [comment]
> BTW, the mill should also have a warning, as it has no road. Fixed that on #211.

Thanks for clarifying that the road is a must have.
